GeeksCoreLibrary/Components/Configurator/Models/VueStepDataModel.cs

```diff
@@ -104,8 +124,23 @@ public class VueStepDataModel
     [JsonProperty("valueDisplayName")]
     public string CurrentValueDisplayName { get; set; }
 
+    /// <summary>
+    /// Gets or sets extra data which are retrieved from the database.
+    /// </summary>
+    [JsonProperty("extraData")]
+    public IDictionary<string, JToken> ExtraData = new ConcurrentDictionary<string, JToken>();
```

Comment:
Why did you use a `ConcurrentDictionary` here? A normal `Dictionary` would suffice here. `ConcurrentDictionary` is a thread-safe dictionary, but that is only applicable for static properties. Or am I missing something?